### PR TITLE
Fix: Moesif Plugin Version Support and Support

### PR DESCRIPTION
**Pull request by dgilling**
@lena-larionova We see the categories were removed after some customers mentioned they couldn't find the Moesif Kong plugin due to [this PR](https://github.com/Kong/docs.konghq.com/pull/6236). Moesif is a Kong Tech partner and already has contracts in place, joint customers, and validation, so it shouldn't be marked as "unsupported". 

### Description

Fix categories for Moesif plugin 

### Testing instructions

Preview link: 

### Checklist 

- [X ] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

### CODE REVIEW

**Review by lena-larionova — COMMENTED** [review]
We'll need to change a couple of small things, following the conversations with our partners team. Once we revert those, we can merge the compatibility updates.

File: app/_hub/moesif/kong-plugin-moesif/_metadata.yml
Comment (lena-larionova) [inline app/_hub/moesif/kong-plugin-moesif/_metadata.yml:1]: Can you please change the `support_url` on line 16 to your support email?

**Review by d19dotca — APPROVED** [review]
Committed the changes for the original submitter.

**Comment by dgilling** [comment]
@d19dotca merged upstream

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**